build.sbt

```diff
@@ -1955,6 +1956,8 @@ lazy val designer = (project in file("designer/server"))
         "com.github.erosb"               % "everit-json-schema"              % everitSchemaV exclude ("commons-logging", "commons-logging"),
         "org.apache.flink"               % "flink-metrics-dropwizard"        % flinkV               % Test,
         "com.github.tomakehurst"         % "wiremock-jre8"                   % wireMockV            % Test,
+        "com.networknt"                  % "json-schema-validator"           % "1.4.0"              % Test,
```

Comment:
I recall Szymon used https://github.com/everit-org/json-schema. Can we stay with one JSON schema validator? We can pick this one if you think it's better the the Everit's one

Reply:
Szymon probably used everit because we already use it in other places in runtime logic of our components. I used it at the beginning but I realize that is much worse than this validator. This validator produces nice list of errors with json paths whereas everit throws an exception with not too much descriptive result. Where Szymon used everit? I'd rather don't want to change all places where everit was used as it is already a side quest to my original task

Reply:
In `NuTapirSchemaTestHelpers`. Is it something we can easily port to the Networknt validator?

Reply:
I see that `NuTapirSchemaTestHelpers` has the same problems as I have when I tried to use Tapir's `Schema` model. It firstly `markOptionsAsNullable = true` and then Szymon had to `deepDropNullValues`. I'm not sure if I want to dive into this ;-) but ... I can try if you want

Reply:
No, it's not necessary. But maybe let's leave a comment.

Reply:
Ignore my question, I'm working on it

Reply:
Done. Now we have such a nice list of errors:
```
$: required property 'parameters' not found,
$: required property 'nodeType' not found,
$: required property 'definition' not found,
$: required property 'service' not found,
$: required property 'outputName' not found,
$: required property 'output' not found,
$: required property 'expression' not found,
$: required property 'value' not found,
```
in this test. It wasn't too much work at the end.